app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -235,7 +235,7 @@ def content_override
 
     api :GET, "/activation_keys/:id/product_content", N_("Show content available for an activation key")
     param :id, String, :desc => N_("ID of the activation key"), :required => true
-    param :content_access_mode_all, :bool, :desc => N_("Get all content available, not just that provided by subscriptions")
+    param :content_access_mode_all, :bool, :desc => N_("Get all content available, not just that provided by subscriptions"), deprecated: true
```

Comment:
I think apipie also supports `default: true`; can you add that and see if it shows up in the doc?

Reply:
Will add it and see what it shows